src/binder/bind/copy/bind_copy_rdf_graph.cpp

```diff
@@ -80,10 +81,13 @@ std::unique_ptr<BoundStatement> Binder::bindCopyRdfFrom(const parser::Statement&
     auto lSource = std::make_unique<BoundFileScanSource>(std::move(lFileScanInfo));
     auto lTableID = rdfGraphEntry->getLiteralTableID();
     auto lEntry = catalog->getTableCatalogEntry(clientContext->getTx(), lTableID);
-    auto lCopyInfo = BoundCopyFromInfo(lEntry, std::move(lSource), offset, std::move(lColumns),
-        logical_type_vec_t{*LogicalType::SERIAL(), *LogicalType::RDF_VARIANT(),
-            *LogicalType::STRING()},
-        std::vector<bool>{false, false}, nullptr /* extraInfo */);
+    auto lNodeEntry = lEntry->constPtrCast<NodeTableCatalogEntry>();
+    auto ser = expressionBinder.bindExpression(*lNodeEntry->getPrimaryKey()->getDefaultExpr());
+    auto lCopyInfo =
```

Comment:
this line is getting a bit long. Can u split it as
```
auto lColumns =
auto lColmnTypes = 
```

Btw, you probably can remove `columnTypes` from `BoundCopyFromInfo` because now you are passing `columnExprs`

Reply:
I will come back to the columnTypes removal later as there is some more change to serial which might affect this